meta-mbl-distro/recipes-core/packagegroups/packagegroup-mbl-development.bb

```diff
@@ -26,6 +26,7 @@ PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " gobject-introspection"
 PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " e2fsprogs"
 PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " mbed-crypto-test"
 PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " memtester"
+PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " mmc-utils"
```

Comment:
Why do we need this package on the rootfs?

Reply:
We don't. I've just got that there temporarily to check the flash erase block sizes on each platform (that has eMMC).

Reply:
Removed mmc-utils in https://github.com/ARMmbed/meta-mbl/pull/617/commits/b050517701a513a3797a1b562a3e16191671b23f.